aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -632,18 +628,11 @@ public class AwsProperties implements Serializable {
    */
   public static final String LAKE_FORMATION_DB_NAME = "lakeformation.db-name";
 
+  private static final String HTTP_CLIENT_URLCONNECTION_PREFIX = "http-client.urlconnection.";
+  private static final String HTTP_CLIENT_APACHE_PREFIX = "http-client.apache.";
   private String httpClientType;
-  private Long httpClientUrlConnectionConnectionTimeoutMs;
-  private Long httpClientUrlConnectionSocketTimeoutMs;
-  private Long httpClientApacheConnectionAcquisitionTimeoutMs;
-  private Long httpClientApacheConnectionMaxIdleTimeMs;
-  private Long httpClientApacheConnectionTimeToLiveMs;
-  private Long httpClientApacheConnectionTimeoutMs;
-  private Boolean httpClientApacheExpectContinueEnabled;
-  private Integer httpClientApacheMaxConnections;
-  private Long httpClientApacheSocketTimeoutMs;
-  private Boolean httpClientApacheTcpKeepAliveEnabled;
-  private Boolean httpClientApacheUseIdleConnectionReaperEnabled;
+  private final Map<String, String> urlConnectionHttpClientProperties;
```

Comment:
I think we can avoid having 2 maps by storing just fields with prefix `http-client.`